__tests__/hooks/useProfileScreenHook.test.js

```diff
@@ -32,12 +33,10 @@ describe('useProfileScreenHook should run correctly', () => {
 
   it('fetchAnonymousPost should run correctly', async () => {
     const {result} = renderHook(useProfileScreenHook, {wrapper});
-    const spyCache = jest.spyOn(StorageUtils.myAnonymousFeed, 'get').mockResolvedValue('[]');
     const spyApiGet = jest
       .spyOn(ProfileRepo, 'getSelfAnonymousFeed')
       .mockResolvedValue({isSuccess: true, data: []});
     await result.current.fetchAnonymousPost(0, 10);
-    expect(spyCache).toHaveBeenCalled();
     expect(result.current.isLoadingFetchingAnonymousPosts).toBeFalsy();
   });
 });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the new code, it seems like you've removed the cache check that was previously there in the old code. This could potentially lead to unnecessary API calls if the data is already available in the cache. Consider adding back the cache check before making the API call.
```diff
+    const spyCache = jest.spyOn(StorageUtils.myAnonymousFeed, 'get').mockResolvedValue('[]');
     const spyApiGet = jest
       .spyOn(ProfileRepo, 'getSelfAnonymousFeed')
       .mockResolvedValue({isSuccess: true, data: []});
     await result.current.fetchAnonymousPost(0, 10);
+    expect(spyCache).toHaveBeenCalled();
     expect(result.current.isLoadingFetchingAnonymousPosts).toBeFalsy();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
This is only test. so closed